Tests/Functional/Bundle/TestBundle/Resources/config/routing.yml

```diff
@@ -21,3 +21,12 @@ test_param_fetcher_test:
 test_view_response_listener:
     resource: FOS\RestBundle\Tests\Functional\Bundle\TestBundle\Controller\ArticleController
     type:     rest
+
+test_allowed_methods1:
+    path:     /allowed-methods
+    methods: ['GET', 'LOCK']
+
+test_allowed_methods2:
+    path:     /allowed-methods
+    defaults: { _controller: TestBundle:AllowedMethods:index }
```

Comment:
What about adding this to the other route for consistency too? I am sure this is my last remark here. ;)